source/NeutronaWand/NeutronaWand.ino

```diff
@@ -3140,21 +3301,11 @@ void wandBarrelHeatDown() {
 
 void fireStream(CRGB c_colour) {
   switch(WAND_BARREL_LED_COUNT) {
-    case LEDS_60:
-      // nothing.
-    break;
-
     case LEDS_48:
-      /*
-      if(ms_firing_stream_blue.justFinished()) {
-
-        ms_firing_stream_blue.start(2);
-
-        ms_fast_led.start(1);
-      }
-      */
+      // Not yet supported.
```

Comment:
What about this line? This can be reached if the user uses the config menu to change their LED count to 48.